osu.Game/Skinning/DefaultLegacySkin.cs

```diff
@@ -1,16 +1,24 @@
 // Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
+using JetBrains.Annotations;
 using osu.Framework.IO.Stores;
+using osu.Game.Extensions;
 using osu.Game.IO;
 using osuTK.Graphics;
 
 namespace osu.Game.Skinning
 {
     public class DefaultLegacySkin : LegacySkin
     {
+        [UsedImplicitly(ImplicitUseKindFlags.InstantiatedWithFixedConstructorSignature)]
```

Comment:
I think you meant this one on the other constructor with the `SkinInfo` at the beginning right?

Reply:
yeah. way more visible of an oversight on the diff than in IDE...